src/pages/docs/deployments/certificates/replace-certificate.md

```diff
@@ -18,7 +19,3 @@ To replace a certificate, navigate to the certificate details page and click 'Re
 :::figure
 ![](/docs/deployments/certificates/images/replace-certificate-btn.png)
 :::
-
-Select the replacement certificate file and password (if required):
-
-![](/docs/deployments/certificates/images/replace-certificate-dialog.png)
```

Comment:
I removed this because it's just an image of the dialog (which the user would see after clicking the button) and I think the UI is intuitive enough to not require the docs mentioning this. This image existing adds some extra work every time the UI changes.